ui/app/templates/jobs/index.hbs

```diff
@@ -48,7 +52,11 @@
       </div>
       {{#if (not (media "isMobile"))}}
         <div class="toolbar-item is-right-aligned">
-          {{#link-to "jobs.run" data-test-run-job class="button is-primary"}}Run Job{{/link-to}}
+          {{#if (can "run job")}}
+            {{#link-to "jobs.run" data-test-run-job class="button is-primary"}}Run Job{{/link-to}}
+          {{else}}
+            <button data-test-run-job class="button tooltip is-right-aligned" aria-label="You don’t have permission to run jobs" disabled>Run Job</button>
+          {{/if}}
```

Comment:
The duplication is a bummer, but it's also not the worst. It's _right_ at the point where it could be dangerous, but at least the two occurrences are co-located. 

I also wouldn't be opposed to `job-run-button` component.